tests/test_1d_gradient/results_true.dat

```diff
@@ -1,203 +1,203 @@
 # Iterations: 47
 keff:  1.68873E-01
 fluxes:
-1.970213E+00
-4.070833E-01
-2.320673E+00
-5.841409E-01
-1.937412E+00
-4.023670E-01
+1.970212E+00
```

Comment:
due to the split segments change. They split the segment that had the max optical path length. As you can see, not all the tests are corrected suggesting the decision of whether to split the segment with the max optical path length was not consistent with this version of the code.